src/metadata/ObjectTypesMetadata.js

```diff
@@ -272,6 +272,30 @@ module.exports = [
             ]
 		},
 	},
+	{
+		name: 'Plug-in - Financial Institution Connectivity',
+		value: {
+			name: 'Plug-in - Financial Institution Connectivity',
+			type: 'ficonnectivityplugin',
+			prefix: 'customscript_',
+            hasRelatedFiles: true,
+            relatedFiles: [
+                { type:'blankscript' }
+            ]
+		},
```

Comment:
Minor one, check identation please, Maybe there are also more objects types like this in the file.